### PR TITLE
Project Idiyanale milestone 1

### PR DESCRIPTION
**Pull request by woss**
Signed-off-by: Daniel Maricic <daniel@woss.io>

# Milestone Delivery Checklist

- [x] The [milestone-delivery-template.md](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/deliveries/milestone-delivery-template.md) has been copied and updated.
- [x] The [invoice form :pencil:](https://forms.gle/LSRr7PCjBpEbKGh89) has been filled out for this milestone.
- [x] This pull request is being made by the same account as the accepted application.
- [x] In case of acceptance, the payment will be transferred to the BTC/ETH payment address provided in the application.
- [x] The delivery is according to the [Guidelines for Milestone Deliverables](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md).

Link to the application pull request: https://github.com/w3f/Grants-Program/pull/719 < please fill this in with the PR number of your application.

### CODE REVIEW

File: deliveries/anagolay-project-idiyanale-phase-1.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Milestone Delivery :mailbox:
+
+**The [invoice form :pencil:](https://docs.google.com/forms/d/e/1FAIpQLSfmNYaoCgrxyhzgoKQ0ynQvnNRoTmgApz9NrMp-hd8mhIiO0A/viewform) has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md).**
+
+-   **Application Document:** https://github.com/w3f/Grants-Program/pull/719
```
Comment (alxs) [inline deliveries/anagolay-project-idiyanale-phase-1.md:5]:
```suggestion
-   **Application Document:** https://github.com/w3f/Grants-Program/blob/master/applications/anagolay-project-idiyanale-phase-1.md
```

**Review by alxs — CHANGES_REQUESTED** [review]

**Comment by alxs** [comment]
Thank you for the deliver @woss. We'll look into it as soon as possible.

**Review by Noc2 — APPROVED** [review]
Sorry for the delay here. I’m happy to confirm that everything is working and that the milestone is a pass. Thanks for the very good delivery. I will forward your invoice internally.

**Comment by woss** [comment]
Hi @Noc2, just a follow-up, is everything ok with the invoice? If your guys have sent the Milestone 1 payment I didn't receive anything yet, if they didn't do you mind telling me when I can expect the payment?

Thanks

**Comment by takahser** [comment]
@woss I'm sorry to hear that, I asked our finance team to double-check. Somebody will reach out to you soon. Please don't hesitate to ask further questions.

**Comment by RouvenP** [comment]
hi @woss we transferred the payment today. Thanks!

**Comment by woss** [comment]
@RouvenP got it :) thanks

**Pull request merged**